generator/src/main/java/com/scottlogic/deg/generator/generation/fieldvaluesources/DateTime/TemporalFieldValueSource.java

```diff
@@ -1,5 +1,6 @@
-package com.scottlogic.deg.generator.generation.fieldvaluesources;
+package com.scottlogic.deg.generator.generation.fieldvaluesources.DateTime;
```

Comment:
Package name should be lowercase

Reply:
I'll address this in the next PR (rename temporal to datetime), i've already done work in this area on that PR, and doing it on this PR will mess that one up

Reply:
Can you cherry-pick the commit across to here, would be best if this PR didn't leave the codebase in a poor state

Reply:
its really much more awkward that i think it's worth.
The name has changed as well as the package, and some of the things in it.